attributes/default.rb

```diff
@@ -99,4 +99,5 @@
 default["kagent"]["certs"]["host_key"]                  = "#{node["kagent"]["certs_dir"]}/priv.key"
 default["kagent"]["certs"]["elastic_admin_key"]         = "#{node["kagent"]["certs_dir"]}/elastic_admin.key"
 default["kagent"]["certs"]["elastic_admin_certificate"] = "#{node["kagent"]["certs_dir"]}/elastic_admin.pem"
-default["kagent"]["certs"]["elastic_admin_cn"]          = node['install']['localhost'].casecmp?("true") ? "ELkAdmin-localhost" : "ELkAdmin-#{node['fqdn']}"
+default["kagent"]["certs"]["elastic_admin_prefix"]      = "ELkAdmin"
```

Comment:
If we don't want to allow overriding it in the cluster-def, then we can hard-code it in line 103?

Reply:
@tkakantousis  I moved into a different attribute to be used in elasticsearch-chef as well 
https://github.com/logicalclocks/elasticsearch-chef/pull/46/files#diff-af441e9eeb4a576380ad3d0def0a4ef0R19

Reply:
Sounds good!